packages/calcite-components/src/components/combobox-item-group/combobox-item-group.tsx

```diff
@@ -53,6 +53,13 @@ export class ComboboxItemGroup extends LitElement {
    */
   @property() scale: Scale = "m";
 
+  /**
+   * Specifies whether the user set the hidden attribute in the HTML
```

Comment:
This comment doesn't seem to be accurate. Can you give it another look? Applies to most `hiddenItem` prop doc.